app/src/test/java/org/mozilla/tv/firefox/pocket/PocketVideoRepoTest.kt

```diff
@@ -21,6 +24,15 @@ import org.robolectric.RobolectricTestRunner
 @RunWith(RobolectricTestRunner::class)
 class PocketVideoRepoTest {
 
+    companion object {
+        @JvmStatic
+        @BeforeClass
+        fun beforeClass() {
+            RxAndroidPlugins.setInitMainThreadSchedulerHandler { Schedulers.trampoline() }
```

Comment:
nit: comment why this is necessary, and if it's not possible to extract this into a Rule, why not